docs/source/tutorial/cli/train.rst

```diff
@@ -60,7 +60,50 @@ Our code supports several methods of splitting data into train, validation, and
 
 * **Scaffold:** Alternatively, the data can be split by molecular scaffold so that the same scaffold never appears in more than one split. This can be specified by adding :code:`--split-type scaffold_balanced`.
 
-* **User Specified Splits** The ability to specify your own split indices will be added soon.
+* **User Specified Splits** Custom splits can be specified in two ways, :code:`--splits-column` and :code:`--splits-file`, examples of which are shown below.
+
+.. code-block::
+
+    chemprop train --splits-column split -i data.csv -t regression
+
+.. list-table:: data.csv
+    :widths: 10 10 10
+    :header-rows: 1
+    
+    * - smiles
+      - property
+      - split
+    * - C
+      - 1.0
+      - train
+    * - CC
+      - 2.0
+      - train
+    * - CCC
+      - 3.0
+      - test
+    * - CCCC
+      - 4.0
+      - val
+    * - CCCCC
+      - 5.0
+      - val
+    * - CCCCCC
+      - 6.0
+      - test
+
+.. code-block::
+
+    chemprop train --splits-file splits.csv -i data.csv -t regression
+
+splits.csv
+
+.. code-block:: JSON
+
```

Comment:
```suggestion
.. code-block:: JSON
    :caption: splits.csv

```

Reply:
![Screen Shot 2024-10-03 at 1 04 59 PM](https://github.com/user-attachments/assets/a65b53c6-7834-4929-848b-98927ece31d5)
Currently, it looks like this. I think we can also put the `splits.csv` as the caption of the code block